### PR TITLE
[base] search word in title on search page

### PR DESCRIPTION
**Pull request by DorienD**
Is this ok? "Search | seach word" or do you want "search word | Search"?

### CODE REVIEW

File: modules/mod_ginger_base/templates/search.tpl [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {% extends "base.tpl" %}
 
-{% block title %}{_ Search _}{% endblock %}
+{% block title %}{_ Search _} | {{ q.qs }}{% endblock %}
```
Comment (ddeboer) [inline modules/mod_ginger_base/templates/search.tpl]: What about `Search: bli bla`? Reads more naturally.

**Comment by krista-ramakers** [comment]
I agree, 'Search: search query' looks good to me!

**Comment by ddeboer** [comment]
Ping @DorienD.

**Comment by ddeboer** [comment]
Why merge this when the comments haven’t been taken into account?

**Comment by DorienD** [comment]
Now it is search: q.qs so that according to the comments right?

**Comment by ddeboer** [comment]
[Nope](https://github.com/driebit/ginger/commit/dc524a1f864aed55565e4b9f19b35b05877fad8c).

**Pull request merged**